resolwe_bio/processes/WGBS/walt.yml

```diff
@@ -137,25 +163,13 @@
       gzip "${NAME}.mr"
       re-progress 0.9
 
-      mv "${NAME}.mr.mapstats" "${NAME}_report.txt"
+      mv "${NAME}.sam.mapstats" "${NAME}_report.txt"
 
-      {% if reads|type|subtype('data:reads:fastq:single:') %}
-        if [ -s "${NAME}.mr_unmapped" ]; then
-          gzip "${NAME}.mr_unmapped"
-          re-save-file unmapped_f "${NAME}.mr_unmapped.gz"
-        fi
-      {% else %}
-        if [ -s "${NAME}.mr_1_unmapped" ]; then
-          gzip "${NAME}.mr_1_unmapped"
-          re-save-file unmapped_f "${NAME}.mr_1_unmapped.gz"
-        fi
-        if [ -s "${NAME}.mr_2_unmapped" ]; then
-          gzip "${NAME}.mr_2_unmapped"
-          re-save-file unmapped_r "${NAME}.mr_2_unmapped.gz"
-        fi
-      {% endif %}
+      gzip "${NAME}_unmapped.bam"
```

Comment:
@jkokosar is this the fastest compression tool we use?

Reply:
Thanks for reminding me, should I use pigz for compression?

Reply:
It's unnecessary to compress the `.bam`, since this is already a compressed binary format of the `.sam` data type.